sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_cbs_async.py

```diff
@@ -171,16 +171,23 @@ async def _update_status(self):
             is_expired, is_refresh_required = check_expiration_and_refresh_status(
                 self._expires_on, self._refresh_window # type: ignore
             )
-            _LOGGER.debug(
-                "CBS status check: state == %r, expired == %r, refresh required == %r",
-                self.auth_state,
-                is_expired,
-                is_refresh_required,
-                extra=self._network_trace_params
-            )
             if is_expired:
+                _LOGGER.debug(
+                    "CBS status check: state == %r, expired == %r, refresh required == %r",
```

Comment:
if we know this is expired, would it be better to have more custom logging here, like: `CBS is expired: state..., refresh required...`?

Reply:
same thing for refresh required below.